operations/node_management/Removing_a_Liquid-cooled_blade_from_a_System_Using_SAT.md

```diff
@@ -105,6 +105,43 @@ This procedure will remove a liquid-cooled blade from an HPE Cray EX system.
 
 ## Use SAT to remove the blade from hardware management
 
+1. Clear out the existing Redfish event subscriptions from the BMCs on the blade.
+
+    1. (`ncn#`) Set the environment variable `SLOT` corresponding to the blades location:
+
+        ```bash
+        SLOT=x9000c3s0
+        ```
+
+    1. (`ncn#`) Clear the Redfish event subscriptions:
+
+        ```bash
+        for BMC in $(cray hsm inventory  redfishEndpoints list --type NodeBMC --format json | jq .RedfishEndpoints[].ID -r | grep $SLOT); do
+            PASSWD=$(cray scsd bmc creds list --targets $BMC --format json | jq .Targets[].Password -r)
+            SUBS=$(curl -sk -u root:$PASSWD https://${BMC}/redfish/v1/EventService/Subscriptions | jq -r '.Members[]."@odata.id"')
```

Comment:
Would be better to avoid putting the password on the curl command line, because that is visible in the output of `ps -ef` for any users logged into the system, regardless of access level.